Tests/DiligentToolsTest/assets/RenderStates/RenderStatePackager/GraphicsPrimitivesDump.json

```diff
@@ -0,0 +1,123 @@
+{
+    "Shaders": [
+        {
+            "Desc": {
+                "Name": "BlitTexture VS",
+                "ShaderType": "VERTEX"
+            },
+            "SourceLanguage": "HLSL",
+            "FilePath": "GraphicsPrimitives.hlsl",
+            "EntryPoint": "VSBlitTexture",
+            "UseCombinedTextureSamplers": true
+        },
+        {
+            "Desc": {
+                "Name": "BlitTexture PS",
+                "ShaderType": "PIXEL"
+            },
+            "SourceLanguage": "HLSL",
+            "FilePath": "GraphicsPrimitives.hlsl",
+            "EntryPoint": "PSBlitTexture",
+            "UseCombinedTextureSamplers": true
+        },
+        {
+            "Desc": {
+                "Name": "ClearUnorderedAccessViewUint CS",
+                "ShaderType": "COMPUTE"
+            },
+            "SourceLanguage": "HLSL",
+            "FilePath": "GraphicsPrimitives.hlsl",
```

Comment:
It is better to use some descriptive name, e.g. BrokenShader.hlsl and don't have any actual code in it (ClearUnorderedAccessViewUint does not add anything), e.g.

BrokenShader.hlsl
`Broken shader code;`

Also, why do you need multiple pipelines in the JSON. It looks like single pipeline will be enough.

Reply:
We need several pipeline descriptions to test the dump with different types of pipelines, and also to make sure that the dump works for all pipelines of the same type.